modules/dynamic-plugin-api.adoc

```diff
@@ -1257,25 +1258,20 @@ A basic lazy loaded YAML editor with hover help and completion.
 |===
 |Parameter Name |Description
 |`value` |String representing the yaml code to render.
-
+|`language` |String representing the language of the editor.
 |`options` |Monaco editor options.
-
+//For more details, please, visit
+//link:https://microsoft.github.io/monaco-editor/api/interfaces/monaco.editor.IStandaloneEditorConstructionOptions.html[Link giving 404].
 |`minHeight` |Minimum editor height in valid CSS height values.
-
 |`showShortcuts` |Boolean to show shortcuts on top of the editor.
-
-|`toolbarLinks` |Array of ReactNode rendered on the toolbar links
-section on top of the editor.
-
+|`toolbarLinks` |Array of ReactNode rendered on the toolbar links section on top of the editor.
 |`onChange` |Callback for on code change event.
-
 |`onSave` |Callback called when the command CTRL / CMD + S is triggered.
-
-|`ref` |React reference to `{ editor?: IStandaloneCodeEditor }`. Using
-the `editor` property, you are able to access to all methods to control
-the editor.
+|`ref` |React reference to `{ editor?: IStandaloneCodeEditor }`. Using the `editor` property, you are able to access to all methods to control the editor.
```

Comment:
Minor suggestion: I cut a few words in the second sentence. Also, "all methods" seemed unquantifiable, so I changed it to "all available methods" to put a boundary around it.

Suggestion: "Using the `editor` property, you can access all available methods to control the editor.

Reply:
These are API's straight from https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/docs/api.md. We would treat these like the REST API docs.